### PR TITLE
Recently edited

### PR DESCRIPTION
**Pull request by jessyrosy**

### CODE REVIEW

File: content/blog/Benefits-of-api-integration/Benefits-of-api-integration.md [hunk 1]
```diff
@@ -0,0 +1,35 @@
+---
+title: "Benefits of API Integration to Developers"
+date: 2022-09-09T10:48:25+01:00
+---
+![benefit-of-api](api-example.jpg)
```
Comment (tonyalaribe) [inline content/blog/Benefits-of-api-integration/Benefits-of-api-integration.md:5]: This should be 
```suggestion
![benefit-of-api](./api-example.jpg)
```

File: content/blog/Benefits-of-api-integration/Benefits-of-api-integration.md [hunk 2]
```diff
@@ -0,0 +1,35 @@
...
+There have been a lot of discussions lately about the critical role APIs play in enhancing any web based activity
+
+Although the developer's perspective has not received as much attention, APIs are also very helpful to almost every set of specialists. They allow developers to work with stakeholders to improve agency systems, creating more potential.
```
Comment (tonyalaribe) [inline content/blog/Benefits-of-api-integration/Benefits-of-api-integration.md:8]: Tautology: Maybe try: APIs are very helpful to every set of specialists.
```suggestion
Although the developer's perspective has not received as much attention, APIs are very helpful to every set of specialists. They allow developers to work with stakeholders to improve agency systems, creating more potential.
```

File: content/blog/Benefits-of-api-integration/Benefits-of-api-integration.md [hunk 3]
```diff
@@ -0,0 +1,35 @@
...
+
+APIs also known as Application Programming Interfaces have been around for a while and have established themselves as crucial technology in numerous industries. APIs enable various kinds of technology to communicate with each other more quickly and simply, from the IT industry to users in the government. Right now we can say that APIs are now more crucial than ever, with big data revealing a lot of information as new technologies emerge. This is a wonderful time to integrate your business with an API because of the advantages that APIs provide.
```
Comment (tonyalaribe) [inline content/blog/Benefits-of-api-integration/Benefits-of-api-integration.md:10]: Could you simplify this article? It's a bit difficult to understand the point of the article. Kinda like there's a lot of filler phrases just to make it longer.

File: content/blog/Benefits-of-api-integration/Benefits-of-api-integration.md [hunk 4]
```diff
@@ -0,0 +1,35 @@
...
+
+## Benefits of API use for developers 
+API integration can cut costs and time to market by up to 90%. Apart from cutting cost this integration has a lot of potential benefits for you. Here, we've outlined the main advantages of integrating APIs into your company.
+- Quick Automation
```
Comment (tonyalaribe) [inline content/blog/Benefits-of-api-integration/Benefits-of-api-integration.md:14]: How do APIs really create Quick Automation? 
If you're integrating an API into an application, someone else needs to have created that API, and you would usually be integrating for a purpose, eg they are offering you a service or data that you need. Eg if you're connecting to a covid19 database API to get the total number of infected people per country. Someone gathered and exposed that data.

File: content/blog/Benefits-of-api-integration/Benefits-of-api-integration.md [hunk 5]
```diff
@@ -0,0 +1,35 @@
...
+
+- API enhances client services
+  With some platforms now offering more than 100 different services accessible through an API alone, APIs also improve the services you provide to your clients. It implies that you can give your clients more value while saving time and money on technical support expenses. Additionally, it implies that APIs are becoming more crucial in the world of eCommerce. By providing better services, you can compete with more established businesses.For instance, if your organization already provides exceptional delivery services, upgrading with an API will boost the value clients perceive in your business.
```
Comment (tonyalaribe) [inline content/blog/Benefits-of-api-integration/Benefits-of-api-integration.md:24]: How do APIs also improve the services you provide your clients?

And `It implies that you can give your clients more value while saving time and money on technical support expenses.` How?

`if your organization already provides exceptional delivery services, upgrading with an API will boost the value clients perceive in your business.` How?

File: content/blog/Benefits-of-api-integration/Benefits-of-api-integration.md [hunk 6]
```diff
@@ -0,0 +1,35 @@
...
+
+- API enhances the marketing procedure
+  APIs can benefit your marketing efforts in a variety of ways, from improved service to more creative strategies. As an illustration, using an API to execute a customer survey might save time and money. Instead of relying on email updates and website content, your clients might complete the survey using any device.
```
Comment (tonyalaribe) [inline content/blog/Benefits-of-api-integration/Benefits-of-api-integration.md:27]: The device is not the API though.  It's not clear to me how APIs actually enhance marketting procedure.

File: content/blog/Benefits-of-api-integration/Benefits-of-api-integration.md [hunk 7]
```diff
@@ -0,0 +1,35 @@
...
+  Additionally, it implies that clients who wish to provide feedback after a survey may do so using their preferred device.
+
+## Simple Tutorials to interating your API with API Toolkit
```
Comment (tonyalaribe) [inline content/blog/Benefits-of-api-integration/Benefits-of-api-integration.md:30]:
```suggestion
## Simple Tutorial for integrating your APIs with APIToolkit
```

Comment (tonyalaribe) [reply content/blog/Benefits-of-api-integration/Benefits-of-api-integration.md:30]: If you're going to give a walkthrough, that's likely better in another post, and you shouldn't just give a list. You should have pictures or screenshots to guide your readers through the process.

File: content/docs/Quickstarts/Golang.md [hunk 8]
```diff
@@ -1,14 +1,14 @@
 ---
 title: Golang 
-date: 2022-03-23
-publishdate: 2022-03-24
+date: 2022-09-09
+publishdate: 2022-09-09
```
Comment (tonyalaribe) [inline content/docs/Quickstarts/Golang.md:4]: I think you should not update publishdata for already existing pages

File: content/docs/Quickstarts/Golang.md [hunk 9]
```diff
@@ -1,14 +1,14 @@
...
 weight: 20
 menu:
   main:
     weight: 20
 ---
 
-The API Toolkit golang client is an sdk used to integrate golang web services with APIToolkit. It monitors incoming traffic, gathers the requests and sends the request to the apitoolkit servers.
+To integrate golang web services with API Toolkit, an SDK called the golang client for API Toolkit is utilized. It keeps track of incoming traffic, aggregates the requests, and then delivers them to the apitoolkit servers.We'll concentrate on providing a step-by-step instruction for integrating an API toolkit into our Golang web service in this tutorial.
```
Comment (tonyalaribe) [inline content/docs/Quickstarts/Golang.md:11]:
```suggestion
To instrument golang web services with API Toolkit, we use the APIToolkit golang SDK. It keeps track of incoming traffic, aggregates the requests, and then delivers them to the apitoolkit servers.We'll concentrate on providing step-by-step instructions for integrating APIToolkit into a Golang web service in this tutorial. 
```

File: content/docs/Quickstarts/Golang.md [hunk 10]
```diff
@@ -17,8 +17,10 @@ The API Toolkit golang client is an sdk used to integrate golang web services wi
   2. Users to explore their API live, via the api log explorer.
 
 ## How to Integrate with Gin router:
-1. Generate the APIToolkit API key 
-2. Initialize the middleware with the APItoolkit API key like you see below:
+1. First things first: Before integrating, always ensure that your codes are functioning properly.
```
Comment (tonyalaribe) [inline content/docs/Quickstarts/Golang.md:20]: I think this line is unnecessary

File: content/docs/Quickstarts/Golang.md [hunk 11]
```diff
@@ -17,8 +17,10 @@ The API Toolkit golang client is an sdk used to integrate golang web services wi
...
+2. Sign up / Sign in to the API dashboard
```
Comment (tonyalaribe) [inline content/docs/Quickstarts/Golang.md:21]: Shouldnt you have a page or proper walkthrough with screenshots about the process

File: content/docs/Quickstarts/Golang.md [hunk 12]
```diff
@@ -17,8 +17,10 @@ The API Toolkit golang client is an sdk used to integrate golang web services wi
...
+3. Create a project, generate your API key, and include a brief description of your work. And to prevent losing your key after it has been generated, remember to make a copy of it.
```
Comment (tonyalaribe) [inline content/docs/Quickstarts/Golang.md:22]: You should  be showing detailed walkthrough.

File: content/docs/Quickstarts/Golang.md [hunk 13]
```diff
@@ -46,3 +48,5 @@ func main() {
 }
 ```
 3. Deploy your application and enjoy exploring and managing your API via the APIToolkit dashboards and API log explorer
+
+Now this is what your dashboard should look like after a successful integration
```
Comment (tonyalaribe) [inline content/docs/Quickstarts/Golang.md:52]: Like what? There's no screenshot

Comment (jessyrosy) [reply content/docs/Quickstarts/Golang.md:52]: There was a screenshot here, that's why you have this here.

File: content/docs/Quickstarts/Golang.md (hunk 11 above)
Comment (jessyrosy) [reply content/docs/Quickstarts/Golang.md:21]: I had screenshots here, but deleting,  installing and all of that process made me lose it. so I just wanted to be sure I am on the right path.

File: content/docs/Quickstarts/Golang.md (hunk 10 above)
Comment (jessyrosy) [reply content/docs/Quickstarts/Golang.md:20]: Okay, I will take it off

File: content/docs/Quickstarts/Golang.md (hunk 8 above)
Comment (jessyrosy) [reply content/docs/Quickstarts/Golang.md:4]: okay

File: content/blog/Benefits-of-api-integration/Benefits-of-api-integration.md (hunk 3 above)
Comment (jessyrosy) [reply content/blog/Benefits-of-api-integration/Benefits-of-api-integration.md:10]: Okay, I will work on it again and let you know when I am done.

**Pull request merged**